### DISCUSSION TITLE
posts/2020/05/battle-ios-email-heavyweights/

### DISCUSSION THREAD

**Discussion by giscus** [brycewray/comments#15] [Announcements]
# posts/2020/05/battle-ios-email-heavyweights/

AltaMail vs. Preside: there’s no MailMate for iOS, but these try to play in the same ballpark.

https://www.brycewray.com/posts/2020/05/battle-ios-email-heavyweights/

**Comment by bbp604** [comment]
The one thing that most email apps do not have is the ability to create extensive mail rules.  This is where one of AltaMail's strengths is in the very competitive email app world.  Combine that with strong privacy and security (mail is stored locally), a very responsive support team (well under a day, sometimes just minutes), and a commitment to continually improve the app give this app high marks in my book.  Is it perfect?  No, but I've never found a solution that is 100%.  There are few idiosyncrasies that could be improved such as attaching files  and searching for mail. Both of these are also UX/UI irritants like how it handles nested folders.  However, overall, these are not dealbreakers.
